### PR TITLE
chore: bump dependencies via dependabot

### PR DESCRIPTION
**Pull request by smira**
See #3431 #3432 #3433 #3434

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -41,7 +41,7 @@ require (
 	github.com/gdamore/tcell/v2 v2.2.0
 	github.com/gizak/termui/v3 v3.1.0
 	github.com/gogo/googleapis v1.4.0 // indirect
-	github.com/golang/protobuf v1.5.1
+	github.com/golang/protobuf v1.5.2
```
Comment (AlekSi) [inline go.mod:44]: The whole module is deprecated now. Does it pass `make lint`?

**Review by AlekSi — APPROVED** [review]

**Review by Unix4ever — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (smira) [reply go.mod:44]: yep, I think there's some legacy we need eventually get rid of to remove this one

**Pull request merged**